docs/reference/pages/panels.rst

```diff
@@ -243,6 +243,24 @@ SnippetChooserPanel
     See :ref:`snippets` for more information.
 
 
+HelpPanel
+---------
+
+.. module:: wagtail.admin.edit_handlers
+
+.. class:: HelpPanel(content, template)
+
+    .. attribute:: HelpPanel.content
+
+        This is string HTML that you can pass in directly to render inside the HelpPanel
```

Comment:
"string HTML" -> "an HTML string", perhaps?

Reply:
I would opt for a simpler version, like:
`HTML string displayed in the panel.`